base/threading/platform_thread.h

```diff
@@ -123,6 +119,10 @@ enum class ThreadType : int {
   // Suitable for low-latency, glitch-resistant audio.
   kRealtimeAudio,
   kMaxValue = kRealtimeAudio,
+  // Default type. The thread priority or quality of service will be set to
+  // platform default. In Chrome, this is suitable for handling user
+  // interactions (input), only display and audio can get a higher priority.
+  kDefault = kSbThreadNoPriority,
```

Comment:
I don't have enough context on the thread priority change, but shall we gate this with a macro, eg: COBALT_PENDING_CLEAN_UP?

Reply:
I think your logic is correct. The Starboard API defined kSbThreadNoPriority as the default priority on each platform, which should be close to Chrome's definition for kDefault.

But I think the simple casting in platform_thread_starboard.cc is very dangerous. Can we keep this file intact but do explicit Chromium->Starboard priority mapping in platform_thread_starboard.cc?

Reply:
Added switch statement with mapping for new ordering of priorities